zkevm-circuits/src/evm_circuit/op_execution/pop.rs

```diff
@@ -0,0 +1,281 @@
+//use crate::evm_circuit::{FixedLookup, Lookup};
+use super::super::{
+    Case, Cell, Constraint, CoreStateInstance, ExecutionStep,
+};
+use super::{CaseAllocation, CaseConfig, OpExecutionState, OpGadget};
+use bus_mapping::evm::{GasCost, OpcodeId};
+use halo2::plonk::Error;
+use halo2::{arithmetic::FieldExt, circuit::Region};
+use std::convert::TryInto;
+use crate::util::Expr;
+
+#[derive(Clone, Debug)]
+struct PopSuccessAllocation<F> {
```

Comment:
If there's only one cell in `PopSuccessAllocation`, we can remove this struct def

Reply:
removeed